sources/HUBComponentImageDataImplementation.m

```diff
@@ -50,7 +50,7 @@ - (instancetype)initWithIdentifier:(nullable NSString *)identifier
         _URL = [URL copy];
         _placeholderIcon = placeholderIcon;
         _localImage = localImage;
-        _customData = customData;
+        _customData = [customData copy];
```

Comment:
If you're `copy`ing here, should the property be marked with `copy` in the header?